### PR TITLE
fix(manager-react-components): update migration guide version

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
ref: MANAGER-14622

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix 14622
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

MRC update documentation

### CODE REVIEW

**Review by tristanwagner — APPROVED** [review]

**Review by darsene — DISMISSED** [review]

**Review by fredericvilcot — APPROVED** [review]

File: packages/manager-react-components/CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,22 @@
...
+- Udpate ods version to 18.3.0
+- Move ods to peerDependencies
+- Update ODS version in Generator
+- Pre-build documentation
+
+# [2.1.0](https://github.com/ovh/manager/commit/377f4206c7e9a4f43aec27b25a1a617e62aa5dba) (2024-11-13)
+
+- Upgrade ods version 18
+- Update generator
+- Fix bump ods 18 issues
```
Comment (anooparveti) [inline packages/manager-react-components/CHANGELOG.md:17]: Can you please mention the actual commit details like https://github.com/ovh/manager/blob/62239840182bd3c5c99a31b37d68731555396e31/packages/manager-react-components/CHANGELOG.md#1410-alpha9-2024-11-08? This makes it easy to look for the commit through the commit history.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager-react-components/CHANGELOG.md (hunk 1 above)
Comment (aboungnaseng-ovhcloud) [reply packages/manager-react-components/CHANGELOG.md:17]: sure !

File: packages/manager-react-components/CHANGELOG.md [hunk 2]
```diff
@@ -3,9 +3,29 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [1.41.1](https://github.com/ovh/manager/compare/@ovh-ux/manager-react-components@1.41.0...@ovh-ux/manager-react-components@1.41.1) (2024-11-13)
+## [2.2.0](https://github.com/ovh/manager/compare/@ovh-ux/manager-react-components@2.1.0...@ovh-ux/manager-react-components@2.2.0) (2024-11-15)
 
-**Note:** Version bump only for package @ovh-ux/manager-react-components
+- Udpate ods version to 18.3.0 ([395ab60](https://github.com/ovh/manager/commit/395ab60fe2ac45f59eed9b54f1fa2c326173a548))
+- Move ods to peerDependencies ([be434c4](https://github.com/ovh/manager/commit/be434c4f3f60b37adf64f90fdce0e827fe61d885))
+- Update ODS version in Generator ([035a111](https://github.com/ovh/manager/commit/035a11128365638999ce0943d01ca8fe56d8d8f8))
+- Pre-build documentation ([b419872](b419872b2f58ef18f8ffb78cc0f5db3218142a3f))
```
Comment (anooparveti) [inline packages/manager-react-components/CHANGELOG.md:11]:
```suggestion
### BREAKING CHANGE

- **manager-react-components:** bump ODS V18 (#14035) ([377f420](https://github.com/ovh/manager/commit/377f4206c7e9a4f43aec27b25a1a617e62aa5dba))
```

File: packages/manager-react-components/CHANGELOG.md [hunk 3]
```diff
@@ -3,9 +3,28 @@
...
+
+### Bug Fixes
+
+- Fix bump ods 18 issues ([72d937f](https://github.com/ovh/manager/commit/72d937f20792d1c668f43dd7cc023c80f93cf1a0))
+
+# [1.41.1](https://github.com/ovh/manager/compare/@ovh-ux/manager-react-components@1.41.0...@ovh-ux/manager-react-components@1.41.1) (2024-11-13)
+
+### Bug Fixes
+
+- Version bump only for package @ovh-ux/manager-react-components ([b86394f](https://github.com/ovh/manager/commit/b86394f1d186821084237348f2001fa930fee0f8))
```
Comment (anooparveti) [inline packages/manager-react-components/CHANGELOG.md:27]: @aboungnaseng-ovhcloud we introduced the breaking change for 2.1.0. Not 2.2.0.
My previous commit was just a sample and it was not good commits. You need to change it with right commits.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by anooparveti — CHANGES_REQUESTED** [review]
i have updated the changelog in #14316. Can you exclude it in this PR?

**Pull request closed without merging**